src/ngraph/pass/manager.cpp

```diff
@@ -111,7 +111,7 @@ void pass::Manager::run_passes(shared_ptr<Function> func, bool transitive)
                 // This checks is to skip the graph optimization when the graph pass relies on static shape
                 // but the function state is dynamic.
                 // we update the function dynamic state only if we run the graph pass successfully.
-                if (function_pass->get_property(pass::PassProperty::REQUIRE_STATIC_SHAPE) &&
+                if (function_pass->get_property(PassProperty::REQUIRE_STATIC_SHAPE) &&
```

Comment:
This wasn't part of my PR. Think there may have been a problem in the cherry-pick here.